### PR TITLE
Add collapsible headers to lesson, script and course editor pages

### PR DESCRIPTION
**Pull request by dmcavoy**
Turns each header on lesson, script and course editor pages into a collapsible header. 

collapse-header

### Updated Course Edit Page
course-collapse

### Updated Script Edit Page
script-collapse

### Update Lesson Edit Page
lesson-collapse

## Links

- [spec](https://docs.google.com/document/d/1LVwA7VtxkkIFZSi7StwAtgHfxFUIAN1E3nbDRjEXWP4/edit#heading=h.3lytjaku1623)
- [jira](https://codedotorg.atlassian.net/browse/PLAT-271)

## Testing story

Added tests for new CollapsibleEditorSection component.
Updated tests to check for new CollapsibleEditorSection component on lesson, script and course editor pages.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by tess323** [comment]
I think this looks great! The only section that while open feels kind of long is Activities, but I think that is a little inevitable with all of the information in there.  Unrelated - on the teachers resources section I think we may want to add some space between resources to visually differentiate, but I can add note that elsewhere.

**Comment by dmcavoy** [comment]
@tess323 The activities section actually has collapsing already built in so that being the really long section works out I guess :)

Definitely agree that the teacher resources section could use some love. Thats an existing component that needs some work but I would hold off on fixing it until we decide if we are using the new Resources model for that.

**Review by tess323 — APPROVED** [review]
Perfect!

File: apps/src/lib/levelbuilder/CollapsibleEditorSection.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    title: PropTypes.string,
+    collapsed: PropTypes.bool,
+    children: PropTypes.any
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      collapsed: this.props.collapsed || false //Default to open if nothing provided
```
Comment (davidsbailey) [inline apps/src/lib/levelbuilder/CollapsibleEditorSection.jsx]: nit: `// Default`... (space after comment symbol)

File: apps/src/lib/levelbuilder/CollapsibleEditorSection.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  editors: {
+    padding: 10
+  }
+};
+
+export default class CollapsibleEditorSection extends Component {
+  static propTypes = {
+    title: PropTypes.string,
+    collapsed: PropTypes.bool,
+    children: PropTypes.any
```
Comment (davidsbailey) [inline apps/src/lib/levelbuilder/CollapsibleEditorSection.jsx:21]: great solution with `children`! Defaulting to expanded sounds right too. 👍

**Review by davidsbailey — APPROVED** [review]

**Review by uponthesun — APPROVED** [review]
This looks awesome!

**Pull request merged**